plugins/gnostic-go-generator/main.go

```diff
@@ -74,3 +77,15 @@ func main() {
 	err = errors.New("No generated code surface model is available.")
 	env.RespondAndExitIfError(err)
 }
+
+func resolvePackageName(p string) (string, error) {
+	p, err := filepath.Abs(p)
```

Comment:
Can you add some comments in here to describe the resolution behavior?

Reply:
how's that?

Reply:
I'm not on my normal computer, so I'm just using the GH editor. If you want me to squash the commits I can do that tonight.

Reply:
I'll squash them when I merge, thanks though!